lambdas/handlers/login_redirect_handler.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import os
+import time
+
+import boto3
+from botocore.exceptions import ClientError
+from oauthlib.oauth2 import WebApplicationClient, InsecureTransportError
+
+from utils.get_aws_region import get_aws_region
+from utils.lambda_response import ApiGatewayResponse
+from services.dynamo_service import DynamoDBService
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def lambda_handler(event, context):
+    return prepare_redirect_response(WebApplicationClient)
+
+
+def prepare_redirect_response(web_application_client_class):
+    try:
+        ssm_response = get_ssm_parameters()
+
+        oidc_parameters = {
+            parameter["Name"]: parameter["Value"]
+            for parameter in ssm_response["Parameters"]
+        }
+
+        oidc_client = web_application_client_class(
+            client_id=oidc_parameters["OIDC_CLIENT_ID"],
+        )
+
+        url, headers, body = oidc_client.prepare_authorization_request(
```

Comment:
Headers and Body don't seem to be used

Reply:
prepare_authorization_request returns a tuple so we've to assign it.